src/bci_build/package.py

```diff
@@ -1235,6 +1274,100 @@ def generate_disk_size_constraints(size_gb: int) -> str:
 """
 
 
+BASALT_BASE = OsContainer(
+    name="base",
+    pretty_name="Base",
+    package_name="base-image",
+    build_recipe_type=BuildType.KIWI,
+    from_image=None,
+    os_version=OsVersion.BASALT,
+    is_latest=True,
+    package_list=[
+        Package(name=pkg_name, pkg_type=PackageType.BOOTSTRAP)
+        for pkg_name in (
+            "aaa_base",
+            "bash",
+            "ca-certificates",
+            "ca-certificates-mozilla",
+            "coreutils",
+            "cracklib-dict-small",
+            "curl",
+            "filesystem",
+            "glibc-locale-base",
+            "gzip",
+            "netcfg",
+            "ALP-dummy-release",
+            "suse-build-key",
+            # "patterns-alp-base",
+            "tar",
+            "timezone",
+            "zypper",
+            "findutils",
```

Comment:
why findutils?

Reply:
Maybe to work around libsolv: https://bugzilla.suse.com/show_bug.cgi?id=1195633